packages/blueprints-integration/src/api.ts

```diff
@@ -148,6 +149,12 @@ export interface ShowStyleBlueprintManifest extends BlueprintManifestBase {
 		playoutStatus: 'current' | 'next'
 	) => void
 
+	/**
+	 * Allows the blueprint to remove the next part instance if it has become orphaned.
+	 * This call will only be made if the part instance has been orphaned.
+	 */
+	shouldRemoveOrphanedPartInstance?: (context: IRemoveOrphanedPartInstanceContext, partInstance: BlueprintRemoveOrphanedPartInstance) => void
```

Comment:
I think this method would make more sense to return a boolean, rather than to call methods on the context to set the result

Reply:
Changed https://github.com/tv2/tv-automation-server-core/commit/584e649d856b353a1e63dd305b852bbe6b14b702